### ISSUE TITLE
Config files for DCGAN

### ISSUE THREAD

**Issue by rangwani-harsh** [POSTECH-CVLab/PyTorch-StudioGAN#4]
Hi,

Kudos for the awesome work. Is it possible for you to release config files for DCGAN experiments? I have a small GPU and BigGAN takes a lot of time to run.

Thanks
Harsh

**Comment by MINGUKKANG** [comment]
Hi,

Thank you for the compliment.

Now, you can conduct SNDCGAN with projection discriminator, SNDCGAN with 2C loss, and Unconditioned SNDCGAN using the config files (https://github.com/POSTECH-CVLab/PyTorch-StudioGAN/tree/master/configs).

Thank you:)

Minguk Kang

**Comment by rangwani-harsh** [comment]
Thanks for the configuration file. I observed some inconsistencies in the mixed precision training code. According to the pytorch documentation there should be only one scalar object for scaling the loss in case of multiple losses ([link](proj_sngan_cifar32_rel_no-train-2020_08_24_03_06_20.log)) as well but in the code there are two scalar objects. Are you able to reproduce the results with two scaler objects?

**Comment by MINGUKKANG** [comment]
Hi:)

Actually, I didn't identify the performances of mixed-precision training now.

I will check concerns about scalar objects and update the code if there are some problems.

Thank you for your advice. 

sincerely, 

Minguk Kang

**Comment by MINGUKKANG** [comment]
Hi!

I have updated our code, especially unifying the gen_scaler and dis_scaler  into a single scaler object.
Thank you for your advice :)
(contra_biggan_cifar32_hinge_nomixed_precision: 9.868+-0.333)
(contra_biggan_cifar32_hinge_mixed_precision: 10.43947 +- 0.386)

Also, we have modified cuDNN settings for faster training and added a switch of autograd API for debugging, which slightly boosts training speed(python3 main.py ... -rm_API).  

Last, fused optimizers are added (python3 main.py ... -fz_op).

Sincerely, 

Minguk Kang

**Comment by rangwani-harsh** [comment]
Thanks for the work. Really helpful. I think if we can provide a way for other's a script to be able to evaluate models and use them without training them it would really increase the utility of the project. You can host models on pytorch hub.
